doc/release/release_notes.rst

```diff
@@ -4,6 +4,28 @@
 Release Notes
 *************
 
+.. release:: Upcoming
+
+    .. change:: changed
+
+        Added compatibility with python 3.6, 3.7 and 3.8.
```

Comment:
I'd be more specific about what changes were introduced here, like:

```suggestion
        Updated :mod:`command_line`, :mod:`filesystem`, :mod:`package`, :mod:`system` 
        and :mod:`utility` to add compatibility with python 3.6, 3.7 and 3.8.
```

(unless the utility encoding change is unrelated to python3?)

Reply:
Is is, thanks for pointing that out